packages/components/src/menu-group/style.scss

```diff
@@ -1,15 +1,12 @@
-.components-menu-group {
-	width: 100%;
-}
-
 .components-menu-group + .components-menu-group {
 	margin-top: $grid-unit-10;
 	padding-top: $grid-unit-10;
-	border-top: $border-width solid $light-gray-500;
+	border-top: $border-width solid $dark-gray-primary;
 }
 
 .components-menu-group__label {
+	text-transform: uppercase;
```

Comment:
Isn't all-caps considered worse for accessibility?

Reply:
I wasn't aware they were. 

I personally think that for a menu group label, which is decidedly seconary (light gray, smaller font size), it's okay to use it as a visual element, it helps the label also act as a separtor. However I'm open to suggestions, any thoughts @pablohoneyhoney?